spec/report_spec.js

```diff
@@ -1,42 +1,45 @@
-const Report = require('../src/report')
-const context = require('aws-lambda-mock-context')
-const schema = require('iopipe-payload').PAYLOAD_SCHEMA
+const Report = require('../src/report');
+const context = require('aws-lambda-mock-context');
+const schema = require('iopipe-payload').PAYLOAD_SCHEMA;
 // arguments
 const config = {
   clientId: 'foo'
-}
+};
 
-describe('Report creation', function() {
-  it('creates a new report object', function() {
-    expect(typeof (new Report(config, context(), process.hrtime(), []))).toBe('object')
-  })
+describe('Report creation', () => {
+  it('creates a new report object', () => {
+    expect(typeof new Report(config, context(), process.hrtime(), [])).toBe(
+      'object'
+    );
+  });
 
-  it('can take no arguments', function() {
-    expect(typeof (new Report())).toBe('object')
-  })
+  it('can take no arguments', () => {
+    expect(typeof new Report()).toBe('object');
+  });
 
-  it('creates a report that matches the schema', function() {
-    const r = new Report()
+  it('creates a report that matches the schema', () => {
+    const r = new Report();
     function iterateKeys(object) {
-      Object.keys(object).forEach(function(key) {
+      Object.keys(object).forEach(key => {
         // custom metrics array
         if (Array.isArray(schema[key])) {
-          expect(Array.isArray(r.report[key])).toBeTruthy()
+          expect(Array.isArray(r.report[key])).toBeTruthy();
         } else if (typeof schema[key] == 'object') {
-          return iterateKeys(schema[key], true)
+          return iterateKeys(schema[key], true);
         }
-        expect(key in r.report)
-      })
+        /* this needs to be examined */
+        return expect(typeof (key in r.report)).toEqual('boolean');
```

Comment:
@pselle Yes, the original code throws an eslint error, and for good reason? It's not really doing anything? What was the intention here? When set to `.isTruthy` it throws because some are false. May need collab on this